### PR TITLE
Scraper updates

### PR DESCRIPTION
**Pull request by dayweek**
- read GPS coordinates directly without geocoding (works in notebook, script needs to be updated)
- remove Google arrows from UI
- added iso file that was missing GeoGuessr/data/utility/iso3166.csv
- fixed script for creating empty directories
- remove some cache data 
- add gitignore
- added Pipfile and python-version for setting environment with pyenv and pipenv

### CODE REVIEW

File: GeoGuessr/scripts/create_rsv_filepath.py [hunk 1]
```diff
@@ -4,8 +4,13 @@
...
 import pandas as pd 
 
-FOLDER_NAME = 'rsv'
-for k in ['indv', 'pano']:
-    os.mkdir(f'../data/{FOLDER_NAME}_{k}')
-    os.mkdir(f'../data/{FOLDER_NAME}_{k}/train')
-    os.mkdir(f'../data/{FOLDER_NAME}_{k}/unlabeled')
+from pathlib import Path
+
+parent_dir = Path(__file__).parent.parent
```
Comment (WinsonTruong) [inline GeoGuessr/scripts/create_rsv_filepath.py:9]: Much smarter method! Thanks for adding it!

File: GeoGuessr/src/RandomStreetViewParser.py [hunk 2]
```diff
@@ -76,7 +76,13 @@ def hide_elements(self):
...
         
         for (element of elements) {
-        element.setAttribute("hidden","");
+          element.setAttribute("hidden","");
+        } 
+
+        // Remove big arrows
+        elements = document.getElementsByTagName('svg');
+        for (element of elements) {
+          element.parentNode.removeChild(element);
```
Comment (WinsonTruong) [inline GeoGuessr/src/RandomStreetViewParser.py:85]: 🥂

**Review by WinsonTruong — APPROVED** [review]

**Comment by WinsonTruong** [comment]
> * read GPS coordinates directly without geocoding (works in notebook, script needs to be updated)
> * remove Google arrows from UI
> * added iso file that was missing GeoGuessr/data/utility/iso3166.csv
> * fixed script for creating empty directories
> * remove some cache data
> * add gitignore
> * added Pipfile and python-version for setting environment with pyenv and pipenv

@dayweek This is such amazing progress, thank you. I couldn't find the gps coordinates directly after hovering over the HTML for so long so I'm really grateful and impressed you found it! 

Could you save your version of the notebook as a separate notebook for now so I can better compare the notebook difference and so we don't get this merge conflict? Aside from that I'm so excited to merge it 🤩

**Comment by dayweek** [comment]
@WinsonTruong I reverted the notebook so there are no conflicts. `run_scraper_with_gps` is the updated version with coordinates and removed arrows.

**Pull request merged**